src/components/selectlist/selectlist.tsx

```diff
@@ -0,0 +1,73 @@
+import { useState } from 'react';
+import styled from 'styled-components';
+import { Button } from '..';
+import { pxToRem, mediaQuery, spacing } from '../../styles';
+
+type Props = {
```

Comment:
always go from top to button like you design the component: the frame - in our case `StyledList` should be first, then `Title` etc. 

Let's also think about better naming ... not everything needs the `styled` attribute